kernel/Kconfig

```diff
@@ -502,30 +502,19 @@ config ARCH_HAS_CUSTOM_BUSY_WAIT
 
 config SYS_CLOCK_TICKS_PER_SEC
 	int "System tick frequency (in ticks/second)"
-	default 100 if QEMU_TARGET || SOC_POSIX
```

Comment:
why are you removing those?

Reply:
sorry, that's a mistake. will update it now